mix.exs

```diff
@@ -20,15 +20,17 @@ defmodule Fennec.Mixfile do
   end
 
   def application do
-    [extra_applications: [:logger],
+    [extra_applications: [:logger, :runtime_tools],
      mod: {Fennec.Application, []}]
   end
 
   defp elixirc_paths(:test), do: ["lib", "test/helper.ex"]
   defp elixirc_paths(_),     do: ["lib"]
 
   defp deps do
-    [{:ex_doc, "~> 0.14", runtime: false, only: :dev},
+    [{:confex, "~> 2.0.1"},
+     {:mix_docker, "~> 0.3.0"},
```

Comment:
Those commands are really pretty. Also, those docker files are almost from this project (`mix docker.customize`). I have "only" tweaked them to use different base image.